guides/vsf.md

```diff
@@ -53,6 +28,7 @@ To prepare your Vue Storefront app for deployment on the Moovweb XDN, run the fo
 ```
 npm install -g @xdn/cli
 xdn init
+xdn build
```

Comment:
Why?  xdn deploy automatically does this.

Reply:
You're right. I will clean up the `Building and Deploying` section as well